### DISCUSSION TITLE
NO ISSUE. Just wanted to say thanks.

### DISCUSSION THREAD

**Discussion by moonman** [openscopeproject/TrguiNG#97] [General]
Hey, I just wanted to say thanks and this transmission client is pretty amazing. I used to use TransmissionRemote GUI, and just recently tried TrguiNG and it has been flawless. Every complaint that came up in my head I found a setting for within 10 seconds. Tabs is great idea for multiple servers. Scroll finally works on laptops (in never did for me with transgui, not properly at least). And holy crap, there is search now!. I guess the only thing I wish this app had was a different colour icon for stopped and seeding torrents, but that's no big deal since I can always look at the status tab. Cheers!

**Comment by qu1ck** [comment]
I am glad you like it. 

The color choice for seeding and finished torrents is intentional to show that they are 2 substates of the same "completed" state. Maybe a slightly different shade of green will work better for finished ones, I'll play with colors a bit.

And thanks for expressing the thanks :)

**Reply by qu1ck** [reply]
In next version finished torrents will have darker green icon

![image](https://github.com/openscopeproject/TrguiNG/assets/3622532/b358dc9a-2450-4ba4-895e-1d0ea2574904)

**Comment by moonman** [comment]
Sweet! Thank you

**Comment by Devillegi** [comment]
Also wanted to tune in to say thank you. 💕

**Comment by Steve-Brule** [comment]
Same. Absolutely in love with TrguiNG. It was the last piece of the puzzle for me to have all my needs met with transmission. Thank you very much!

**Comment by CitizenJohnJohn** [comment]
Popping in to also say thank you! Huge improvement on TransmissionRemote GUI and great to see the pace of improvememnts and bug fixes. Chapeau!

**Comment by elisimpson** [comment]
It really is outstanding. Thank you kindly!

**Comment by CitizenJohnJohn** [comment]
Thanks for v 1.4!

**Comment by moonraker2** [comment]
Me too, I liked the now-abandoned transgui, but I love TrguiNG :heart: 
Thanks a bunch for the development effort!

**Comment by rheinon** [comment]
Just dropped by to say why I didn't found about this sooner.  This is awesome!

**Comment by StingRaymond** [comment]
I’m using exclusively as web interface, I love this in the web browser
It’s really awesome

Thanks for the great work

**Comment by ayala-io** [comment]
Echoing the comments here! transgui was great, but had little quirks, occasional crashes, and weird flickering issues when I maximize the window. TriguiNG is a fantastic refresh and the true successor. Thank you so much for this!

**Comment by brxxm** [comment]
Last week apple gave a message on my M3 MacBook that rosetta support for intel is to end in the near future. TR GUI seems to be abandoned so I looked for future options. Came here thanks to Google Gemini. Installation was smooth sailing, up and running within minutes. Thank you very much for providing this app for Apple Silicon!
